services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/auth/IAuthUserWithPermissions.java

```diff
@@ -0,0 +1,36 @@
+package com.sourcefuse.jarc.services.usertenantservice.auth;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.Data;
+
+/*class creating for testing purpose not required for user tenant serv*/
+@Data
+public class IAuthUserWithPermissions extends IAuthUser {
```

Comment:
We dont need this

Reply:
ok ,but I have used this to get the current user userId /TenantID etc
do we have this in core lib ?
or which class i should use ?
or do we need remove as of now?